sql/core/src/test/scala/org/apache/spark/sql/execution/PlannerSuite.scala

```diff
@@ -157,4 +157,42 @@ class PlannerSuite extends SparkFunSuite {
     val planned = planner.TakeOrderedAndProject(query)
     assert(planned.head.isInstanceOf[execution.TakeOrderedAndProject])
   }
+
+  test("PartitioningCollection") {
+    // First, we disable broadcast join.
+    val origThreshold = conf.autoBroadcastJoinThreshold
+    setConf(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD, 0)
```

Comment:
Actually, it looks like the implementation might be out of sync w.r.t. the docs for `AUTO_BROADCASTJOIN_THRESHOLD`...